src/cascadia/TerminalApp/CascadiaSettings.h

```diff
@@ -39,9 +39,17 @@ namespace TerminalAppUnitTests
 
 namespace TerminalApp
 {
+    class SettingsTypedDeserializationException;
     class CascadiaSettings;
 };
 
+class TerminalApp::SettingsTypedDeserializationException final : public std::runtime_error
+{
+public:
+    SettingsTypedDeserializationException(const std::string_view description) :
+        runtime_error(description.data()) {}
+};
+

```

Comment:
There's already `TerminalWarnings.h` - is that maybe a better place for this to be defined (I suppose this is a nit)

Reply:
Yeah... it _is_ a better place but I was somewhat expecting that we would come plowing through here with a refactor eventually when we moved it to another dll